invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/Results.js

```diff
@@ -69,6 +69,25 @@ export const Results = ({ currentResultsState = {} }) => {
               />
             </Grid.Column>
           </Grid.Row>
+          <Grid.Row className="mobile only" verticalAlign="middle">
```

Comment:
you can handle grid column sizes with computer= tablet= props instead of "mobile only" class, I think we could avoid the code duplication for the pagination component in this way